library/core/src/char/convert.rs

```diff
@@ -155,6 +156,20 @@ impl From<char> for u128 {
     }
 }
 
+/// Map `char` with code point in U+0000..=U+00FF to byte in 0x00..=0xFF with same value, failing
+/// if the code point is greater than U+00FF.
+///
+/// See [`impl From<u8> for char`](../std/primitive.char.html#impl-From<u8>) for details on the encoding.
```

Comment:
Try `(char)`, or if that doesn't work, more relative paths.
```suggestion
/// See [`impl From<u8> for char`](../../std/primitive.char.html#impl-From<u8>) for details on the encoding.
```

Reply:
> Try `(char)`

```[`impl From<u8> for char`](char)``` works, but of course that doesn't link to the specific trait implementation. Not sure if you mean something else.

Adding an extra `..` fixes the link in the `TryFrom` docs, but breaks the link in the `u8` docs, since the pages are `std/convert/trait.TryFrom.html` and `std/primitive.u8.html`.

Reply:
You want `(char#impl-From<u8>)`.

Reply:
Ah. Doesn't seem to work though:

> error: `char#impl-From<u8>` contains an anchor, but links to builtin types are already anchored
   --> library/core/src/char/convert.rs:162:36
    |
162 | /// See [`impl From<u8> for char`](char#impl-From<u8>) for details on the encoding.
    |                                    ^^^^--------------
    |                                        |
    |                                        invalid anchor
    |
    = note: `-D rustdoc::broken-intra-doc-links` implied by `-D warnings`
    = note: this restriction may be lifted in a future release
    = note: see https://github.com/rust-lang/rust/issues/83083 for more information

So it should work correctly once https://github.com/rust-lang/rust/pull/87073 is merged?

Reply:
Oh, good point. Yes, it should work once that's merged, but that's at least few weeks from now, I haven't gotten initial feedback and I might need a different approach. I would use (char) until then.